apps/src/gamelab/TextConsole.jsx

```diff
@@ -1,10 +1,58 @@
 /** @file Grid over visualization */
 import PropTypes from 'prop-types';
 import React from 'react';
+import {Transition} from 'react-transition-group';
 
 export const styles = {
   hide: {
     display: 'none'
+  },
+  console: {
+    display: '',
+    background: 'rgba(128,128,128,0.3)',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    zIndex: 2,
+    width: '100%',
+    transitionProperty: 'max-height',
+    transitionDuration: '1s'
+  },
+  expandButton: {
+    position: 'absolute',
+    right: '0',
+    zIndex: 3,
+    minWidth: '16px',
+    margin: '0px',
+    border: '0px',
+    padding: '0px',
```

Comment:
Tiny style nit: can use the int `padding: 0` here and elsewhere, to use default units.

Reply:
Fixed. Thanks!